### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by VAIBHAVSINGH2411**

### CODE REVIEW

**Review by speckysourabh — APPROVED** [review]

**Review by speckysourabh — APPROVED** [review]

**Review by speckysourabh — APPROVED** [review]

**Review by speckysourabh — COMMENTED** [review]
import { event } from "jquery";

Remove if this import is not used

**Review by speckysourabh — APPROVED** [review]

**Review by speckysourabh — COMMENTED** [review]
Remove useless comments

**Review by speckysourabh — COMMENTED** [review]
Remove commented code

File: speckyfox-event/src/components/HomePageConfiguration/HomePageConfiguration.jsx [hunk 1]
```diff
@@ -45,9 +45,9 @@ function HomePageConfiguration(props) {
         }
         request.append("banner", banner);
         request.append("logo", logo);
-        for (const entry of request.entries()) {
-          console.log(entry[0], entry[1]);
-        }
+        // for (const entry of request.entries()) {
```
Comment (SpeckySurya) [inline speckyfox-event/src/components/HomePageConfiguration/HomePageConfiguration.jsx]: Please remove commented code

File: speckyfox-event/src/components/HomePageConfiguration/HomePageConfiguration.jsx [hunk 2]
```diff
@@ -45,9 +45,9 @@ function HomePageConfiguration(props) {
...
+        //   console.log(entry[0], entry[1]);
```
Comment (SpeckySurya) [inline speckyfox-event/src/components/HomePageConfiguration/HomePageConfiguration.jsx]: Please remove commented code

File: speckyfox-event/src/components/ManageSpeaker/ManageSpeaker.jsx [hunk 3]
```diff
@@ -41,9 +41,9 @@ function ManageSpeaker(props) {
           request.append(entry, values[entry]);
         }
         request.append("picture", selectedFile);
-        for (let item of request.entries()) {
-          console.log(item[0], item[1]);
-        }
+        // for (let item of request.entries()) {
```
Comment (SpeckySurya) [inline speckyfox-event/src/components/ManageSpeaker/ManageSpeaker.jsx]: Please remove commented code

File: speckyfox-event/src/components/NotifyParticipant/NotifyParticipant.jsx [hunk 4]
```diff
@@ -23,7 +23,7 @@ const NotifyParticipant = () => {
   const [loading, setLoading] = useState(false);
   const [isAlertVisible, setIsAlertVisible] = useState(false);
 
-  console.log(selectedEvent);
+  // console.log(selectedEvent);
```
Comment (SpeckySurya) [inline speckyfox-event/src/components/NotifyParticipant/NotifyParticipant.jsx]: Please remove commented code

**Review by SpeckySurya — CHANGES_REQUESTED** [review]

**Review by speckysourabh — APPROVED** [review]

**Review by SpeckySurya — APPROVED** [review]

**Pull request merged**